pulp_2to3_migration/app/migration.py

```diff
@@ -297,40 +260,36 @@ def create_repoversions_publications_distributions(plan, parallel=True):
     for plugin in plan.get_plugin_plans():
         pulp3_repo_setup = plugin.get_repo_creation_setup()
 
-        if not pulp3_repo_setup:
-            task_func = simple_plugin_migration
-            task_args = [plugin]
-            task_func(*task_args)
-        else:
-            task_func = complex_repo_migration
-            repo_ver_to_create = 0
-            dist_to_create = 0
-            if parallel:
-                for repo_name in pulp3_repo_setup:
-                    repo_versions = pulp3_repo_setup[repo_name]['repository_versions']
-                    repo_ver_to_create += len(repo_versions)
-                    for repo_ver in repo_versions:
-                        dist_to_create += len(repo_ver['dist_repo_ids'])
-                    repo = Repository.objects.get(name=repo_name).cast()
-                    task_args = [plugin, pulp3_repo_setup, repo_name]
-                    enqueue_with_reservation(
-                        task_func,
-                        [repo],
-                        args=task_args,
-                        task_group=TaskGroup.current()
-                    )
-                task_group = TaskGroup.current()
-                progress_rv = task_group.group_progress_reports.filter(code='create.repo_version')
-                progress_rv.update(total=F('total') + repo_ver_to_create)
-                progress_dist = task_group.group_progress_reports.filter(
-                    code='create.distribution'
+        repo_ver_to_create = 0
+        dist_to_create = 0
+
+        if parallel:
+            for repo_name in pulp3_repo_setup:
+                repo_versions = pulp3_repo_setup[repo_name]['repository_versions']
+                repo_ver_to_create += len(repo_versions)
+                for repo_ver in repo_versions:
+                    dist_to_create += len(repo_ver['dist_repo_ids'])
+                repo = Repository.objects.get(name=repo_name).cast()
+                task_args = [plugin, pulp3_repo_setup, repo_name]
+                enqueue_with_reservation(
+                    complex_repo_migration,
+                    [repo],
+                    args=task_args,
+                    task_group=TaskGroup.current()
                 )
-                progress_dist.update(total=F('total') + dist_to_create)
-            else:
-                # Serial (non-parallel)
-                for repo_name in pulp3_repo_setup:
-                    task_args = [plugin, pulp3_repo_setup, repo_name]
-                    task_func(*task_args)
+        else:
+            # Serial (non-parallel)
```

Comment:
is there a reason to leave this piece of code? what would be the benefit of creating repos in serial way from complex plan?

Reply:
It makes testing easier